spec/jobs/mesh_validate_mailbox_job_spec.rb

```diff
@@ -0,0 +1,19 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+
+describe MESHValidateMailboxJob do
+  it "calls MESH.validate_mailbox" do
+    Flipper.enable(:mesh_jobs)
+    allow(MESH).to receive(:validate_mailbox)
```

Comment:
This line could go in a `before` block as it's used in both tests.

Reply:
Amended